diff -Nurb o.vdkbuilder2-2.4.0/configure.in n.vdkbuilder2-2.4.0/configure.in
--- o.vdkbuilder2-2.4.0/configure.in	2004-05-18 14:31:17.000000000 +0200
+++ n.vdkbuilder2-2.4.0/configure.in	2005-08-28 01:40:11.341036792 +0200
@@ -65,8 +65,6 @@
 dnl-------------------
 AC_ARG_ENABLE(devel,    [ --enable-devel[=yes/no]  Compile VDKBuilder with all info for debug [default=no]],,enable_devel=no)
 
-AC_ARG_ENABLE(opt,      [ --enable-opt[=OPTIMIZATION/no]  Compile with -Ox optimization [default=2]],,enable_opt=2)
-
 dnl AC_ARG_ENABLE(gnome,[  --enable-gnome=[yes/no]   Enable use of gnome[default=no], not required if vdk was compiled with gnome-support],
 dnl	gnome=$enableval, gnome=no)
 
@@ -85,22 +83,11 @@
 dnl check user options
 dnl -------------------
 if test "x$enable_devel" = "xyes"; then
-  cflag="-g"
   AC_DEFINE(VDKBDEBUG)
-else
-  cflag="-s"
-fi
-
-if test "x$enable_opt" != "xno"; then
-  cflag="$cflag -O$enable_opt"
-fi
-
-if test "x$prefix" = "xNONE"; then
-  prefix="/usr/local"
 fi
 
-CFLAGS="$cflag -I../include -Wall"
-CXXFLAGS="$cflag -I../include  -Wall"
+CFLAGS="-I../include -Wall $CFLAGS"
+CXXFLAGS="-I../include -Wall $CXXFLAGS"
 
 dnl check vdkxdb support
 if test "x$enable_vdkxdb" == "xyes"; then
